src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -69,19 +66,6 @@ export async function getCowSwapTradeQuote(
   )
 
   if (maybeQuoteResponse.isErr()) {
-    const err = maybeQuoteResponse.unwrapErr()
-    const errData = (err.cause as AxiosError)?.response?.data
-    if (
-      (err.cause as AxiosError)?.isAxiosError &&
-      errData?.errorType === 'SellAmountDoesNotCoverFee'
-    ) {
-      return Err(
-        createTradeAmountTooSmallErr({
-          assetId: sellAsset.assetId,
-          minAmountCryptoBaseUnit: bn(errData?.data.fee_amount ?? '0x0', 16).toFixed(),
```

Comment:
Not present in CoW errors anymore, see https://api.cow.fi/docs/#/default/post_api_v1_quote